env0/resource_aws_credentials.go

```diff
@@ -0,0 +1,106 @@
+package env0
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAwsCredentials() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAwsCredentialsCreate,
+		ReadContext:   resourceAwsCredentialsRead,
+		DeleteContext: resourceAwsCredentialsDelete,
+
+		Importer: &schema.ResourceImporter{StateContext: resourceAwsCredentialsImport},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Description: "name for the credentials",
+				Required:    true,
+			},
+			"arn": {
+				Type:        schema.TypeString,
+				Description: "the aws role arn",
+				Required:    true,
+			},
+			"external_id": {
+				Type:        schema.TypeString,
+				Description: "the aws role external id",
+				Required:    true,
```

Comment:
@roni-frantchi you can make this one `Sensitive: true` as well 💡

Reply:
# 💡

Reply:
https://github.com/env0/terraform-provider-env0/pull/48/commits/c9fd26e9ed6ba64dc67527d9ba566353f3da9f84